topostats/grains.py

```diff
@@ -202,10 +220,16 @@ def area_thresholding(self, image: np.ndarray, area_thresholds: list):
             upper = image.size * self.pixel_to_nm_scaling**2
         if lower is None:
             lower = 0
+        # Get array of grain numbers (discounting zero)
         uniq = np.delete(np.unique(image), 0)
         grain_count = 0
-        for grain_no in uniq:
+        LOGGER.info(
+            f"[{self.filename}] : Area thresholding grains | Thresholds: L:{lower / self.pixel_to_nm_scaling**2:.2f},\
```

Comment:
You can split long log messages by just closing the string then having another quoted string straight after (see `validation.py` for examples). Saves having to escape carriage returns and text remains readable.